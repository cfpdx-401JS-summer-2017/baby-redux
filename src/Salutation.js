import React from 'react';
import { connect } from 'react-redux';
import Receiver from './containers/Receiver';

//const { createStore } = require('redux');

export function Salutation({greeting, name}) {
    return (
        <div>
            <section>
                <span>{greeting}</span>,
                <span>{name}</span>!
            </section>
            <section>
                <Receiver />
            </section>
        </div>
    );
}

export default connect(
    state => ({ greeting: state.greeting , name: state.name})
)(Salutation);