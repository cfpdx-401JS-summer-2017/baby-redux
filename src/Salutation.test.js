import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import { Salutation } from './Salutation';

describe('Counter', () => {

    it('shows a greeting and two inputs', () => {
        const wrapper = shallow(<Salutation />);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});