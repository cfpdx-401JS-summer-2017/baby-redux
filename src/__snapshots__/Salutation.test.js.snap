// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter shows a greeting and two inputs 1`] = `
<div>
  <section>
    <span />
    ,
    <span />
    !
  </section>
  <section>
    <Connect(Receiver) />
  </section>
</div>
`;
